2019/pkg/intcode/intcode.go

```diff
@@ -51,91 +66,67 @@ type Program struct {
 	output         []int
 }
 
-func NewProgram(input []int) *Program {
+func NewProgram(instructions []int) *Program {
```

Comment:
I would call `input / instructions` either `code` or `memory`.

Reply:
are you saying to only rename this parameter? Or to rename the field `instructions` too? Because I did the latter.